### ISSUE TITLE
This wallet is crooked

### ISSUE THREAD

**Issue by alirezasha** [iotaledger/android-wallet-app#169]
Why does not this wallet carry out the transactional transaction? More than a week has passed since the post has been pending

**Comment by mmainstreet** [comment]
Complete new wallet is under development (trinity wallet). So this one will only be maintained for security.

**Comment by alirezasha** [comment]
If so, then why is it pending?

**Comment by Lward85** [comment]
Yeah. My IOTA got hacked. Git screenshots too. Can you track down the receiving person?

**Comment by mmainstreet** [comment]
head over to https://discordapp.com/invite/fNGZXvh. the community will help you

@Lward85 yes, whats the address?
